### DISCUSSION TITLE
Account updater

### DISCUSSION THREAD

**Discussion by cb-sudheer** [juspay/hyperswitch#3163] [Ideas / Feature Requests]
Our system needs to sync with updated card details when changes are made by the bank.

**Comment by bernard-eugine** [comment]
Hey! Account updater is in our list of things under consideration for our Q1'24 roadmap. We will confirm if we are picking this up. 

But we would like to understand how you are managing Account updation currently or if you would be interested in contributing to the integration in case you already have enough context?

**Reply by manojradhakrishnan** [reply]
@cb-sudheer Pls clarify whether the account updater requested here is an integration with Payment processor's account updater API or with Card network API? 

Any specific payment processors in focus?

**Reply by cb-sudheer** [reply]
At present, our setup involves utilizing a Payment processor account updater. However, looking ahead, leveraging the card networks appears to be the optimal choice for the future. We are open to any available option in this regard.

**Comment by luizaraujobliver** [comment]
@manojradhakrishnan Was this integration with card networks prioritized for this quarter?

**Comment by bernard-eugine** [comment]
@luizaraujobliver Hey we haven't picked it up for this quarter as we are focusing on decoupled 3DS auth. You can follow our roadmap here: https://docs.hyperswitch.io/about-hyperswitch/roadmap

If we decide to pick up Account updater in the next quarter, you would be able to follow it in the above link
